setup.py

```diff
@@ -7,15 +7,27 @@
 
 
 this_directory = path.abspath(path.dirname(__file__))
-files = {'Readme': 'README.md',
-         'Changelog': 'CHANGELOG.md'}
+
+files = {'Readme': 'README.md', 'Changelog': 'CHANGELOG.md'}
+
 long_description = ""
 for name, filename in files.items():
-    long_description += "{}\n\n".format(name)
+    long_description += "# {}\n\n".format(name)
     with open(path.join(this_directory, filename)) as f:
         file_contents = f.read()
     long_description += file_contents + "\n\n"
 
+long_description = ""
```

Comment:
This overwrites the value built up by the previous loop?

Reply:
Duplicated code.